lms/static/js/spec/learner_dashboard/course_enroll_view_spec_2017.js

```diff
@@ -0,0 +1,293 @@
+define([
```

Comment:
we already had this file, so this is almost a carbon copy with some small tweaks for the new code